workitem/expression_compiler.go

```diff
@@ -103,6 +103,10 @@ func (c *expressionCompiler) Equals(e *criteria.EqualsExpression) interface{} {
 	return c.binary(e, "=")
 }
 
+func (c *expressionCompiler) IsNull(e *criteria.IsNullExpression) interface{} {
+	return "(Fields->>'" + e.FieldName + "' IS NULL)"
```

Comment:
I could see that we can index particular field on jsonb like show here https://blog.codeship.com/unleash-the-power-of-storing-json-in-postgres/

so, `CREATE INDEX fields_assignee_idx ON WI ((fields->>'assignees'));` will be helpful